apps/portlet-quarantine/src/main/resources/locale/portlet/quarantine_en.properties

```
@@ -0,0 +1,10 @@
+documents.quarantine.label=Quarantine
```

```suggestion
documents.dlp.quarantine.label=Quarantine
```
same for other labels